src/course-outline/page-alerts/PageAlerts.test.jsx

```diff
@@ -56,8 +56,8 @@ describe('<PageAlerts />', () => {
   });
 
   it('renders null when no alerts are present', () => {
-    const { container } = renderComponent();
-    expect(container).toBeEmptyDOMElement();
+    const { queryByTestId } = renderComponent();
+    expect(queryByTestId('browser-router')).toBeEmptyDOMElement();
```

Comment:
As of https://github.com/openedx/frontend-platform/pull/615, the `AppProvider` used in `renderComponent` includes
```html
<div data-testid="browser-router">
```
and the `OptionalReduxProvider` in the `AppProvider` includes
```html
<div data-testid="redux-provider">
```

so instead of the whole container being an empty DOM element, it is
```html
<div data-testid="redux-provider"><div data-testid="browser-router"></div></div>
```

This just updates the test to ensure nothing is rendered inside the divs that are always rendered